CSLabs.Api/Models/Maintenance.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace CSLabs.Api.Models
+{
+    public class Maintenance
+    {
+        public int Id { get; set; }
+        public DateTime? StartTime { get; set; }
+        public DateTime? EndTime { get; set; }
```

Comment:
@WarriorAchilles Why are these marked as nullable? Shouldn't the start and end time of a maintenance be required?